spec/models/lesson_spec.rb

```diff
@@ -59,4 +58,19 @@
       expect(lesson.position_in_section).to eql(2)
     end
   end
+
+  describe '#import' do
+    let(:github_lesson) { FactoryGirl.create(:lesson, url: "/README.md", content: nil) }
```

Comment:
We are in the process of moving away from factory girl in our tests, could you add `url: "/README.md"` and `content: nil` into the lesson double at the top of the and use the `lesson` double in your tests please.